### ISSUE TITLE
The mAP loss is not found.

### ISSUE THREAD

**Issue by mumuner** [shuxjweb/last#4]
Hi, I checked your code but I didn't find the mAP loss. And is your "_base.py" means that the baseline is BoT?

**Comment by shuxjweb** [comment]
Hi, the model in "_bot.py" is BoT.  You can see the model name in "modeling/__init__.py". As the paper is under review, we have not released the mAP loss.

**Comment by mumuner** [comment]
Hi, I had train the last_train_bot.py. When I'm going to test, there is an error:  Tried to allocate 4.75 GiB (GPU 0; 10.76 GiB total capacity; 2.34 GiB already allocated; 3.67 GiB free; 5.93 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF. Can you help me to solve this problem? please.

**Comment by shuxjweb** [comment]
The test dataset is very large. Maybe you can try it with a larger GPU, for example, P40 or V100.

**Comment by mumuner** [comment]
Hi, I had train the last_train_bot.py. The Rank1 and mAP achieve 88.0% and 34.6% on validation set respectively, but the Rank1 and mAP on test set are 67.0% and 23.6% respectively. There is a big gap between the validation set and the test set, can you explain?

**Comment by shuxjweb** [comment]
The scales between validation set and test set are quite different. As the testing time is very long, we provide a smaller one (the validation set) to optimize the model. Please check it carefully.
